### ISSUE TITLE
[Feature Request] Removal of black bar letterboxing durring cutscenes or Z-Targetting

### ISSUE THREAD

**Issue by altheawesomeguy** [Mr-Wiseguy/Zelda64Recomp#300]
Title sums it up. Letterboxing was fine when we all were on 4:3 screens.

![Screenshot at 2024-05-27 16-21-54](https://github.com/Zelda64Recomp/Zelda64Recomp/assets/103470860/a49578ce-997f-4a0b-aa72-69e54e27133e)

**Comment by alby13** [comment]
I think that this would simply be a great option. This is simply black bars that are being put on for "cinematic" reasons, right?

**Comment by FSR-1** [comment]
You want to remove a staple of 3D Zelda

**Comment by NevarcBB** [comment]
Ship of Harkinian has had this option for a while now and I think it makes the presentation more cohesive for 16:9 displays. Would be a nice option to have here.

**Comment by FSR-1** [comment]
> Such a staple that the two most recent games don't have it :P

the two most recent games are a massive departure from normal 3D Zelda and even the remakes kept the letter boxing so I think it should be kept on by default

**Comment by SirYodaJedi** [comment]
I think the amount of letterboxing should be configurable, rather than a simple boolean toggle. The letterboxing when z-targeting *isn't* a purely stylistic decision; it's also there to give visual feedback that the user is z-targeting. 
The widescreen Zelda games still have letterboxing when z-targeting, but it covers significantly less of the screen (resulting in about 2:1 aspect ratio for the 16:9 games, and about 16:9 for the 16:10 games). Some of the games also apply that small amount of letterboxing during cutscenes, but it seems like this isn't the case for the 3DS remasters; those only have letterboxing when z-targeting. 

To sum it up:
* Letterbox/pillarbox settings should be individually configurable for "cutscene" and "z-targeting"
* Either have a slider for each that goes from `-25%` to `25%` controlling letter/pillarbox size, or at least have the ability to select between `N64 (25%)`, `3DS (10%)`, `Preserve original aspect ratio` (see #63), and `Open matte` presets  for each (which scale to the given output aspect ratio)

**Comment by DarioSamo** [comment]
A mod for this is available.
